src/activity_item_handling.cpp

```diff
@@ -1489,6 +1489,11 @@ static activity_reason_info can_do_activity_there( const activity_id &act, playe
                     // do we have the required seed on our person?
                     const auto options = dynamic_cast<const plot_options &>( zone.get_options() );
                     const std::string seed = options.get_seed();
+                    // If its a farm zone with no specified seed, and we've checked for tilling and harvesting.
+                    // then it means no further work can be done here.
+                    if( seed == "No seed" ) {
```

Comment:
Shouldn't we check for something like `itype_id( "null" )` instead of plain string?

Reply:
The zone "options" are setup to store it as "No seed" which is what options.get_seed() outputs as, it dosnt output with an itype_id.  

see in clzones.cpp get_seed() just returns `seed` which is a string of "No seed" when no seed itype has been specified, though I guess that area could be changed,  

`    std::vector<seed_tuple> seed_entries = iexamine::get_seed_entries( seed_inv );
    seed_entries.emplace( seed_entries.begin(), seed_tuple( itype_id( "null" ), _( "No seed" ), 0 ) );`